connect/mirror-client/src/main/java/org/apache/kafka/connect/mirror/DefaultReplicationPolicy.java

```diff
@@ -70,4 +70,32 @@ public String upstreamTopic(String topic) {
             return topic.substring(source.length() + separator.length());
         }
     }
+
+    private String internalSuffix() {
+        return separator + "internal";
+    }
+
+    private String checkpointsTopicSuffix() {
+        return separator + "checkpoints" + internalSuffix();
+    }
+
+    @Override
+    public String offsetSyncTopic(String targetAlias) {
```

Comment:
Should we name the argument `clusterAlias` as it's not necessarily the target cluster alias?